src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    // 26.04.2019 John 4 50
+    private static final int dateSubstring = 0;
+    private static final int nameSubstring = 1;
+    private static final int timesPaidSubstring = 2;
+    private static final int sumPaidSubstring = 3;
```

Comment:
Fix all names.
What is '0'? Is it 'date'? No. '0' is **index** of the array where the date is located. So, DATE_INDEX would be correct name for the constant.